acme-cli/src/main/java/io/micronaut/acme/cli/AcmeServerOption.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2017-2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.micronaut.acme.cli;
+
+import picocli.CommandLine;
+
+/**
+ * Allows for helper flags to make selecting which ACME environment you are using easier.
+ */
+public final class AcmeServerOption {
+
+    public static final String LE_PROD_URL = "https://acme-v02.api.letsencrypt.org/directory";
+
+    public static final String LE_STAGING_URL = "https://acme-staging-v02.api.letsencrypt.org/directory";
+
+    @CommandLine.Option(names = {"-u", "--url"}, required = true, showDefaultValue = CommandLine.Help.Visibility.ALWAYS, description = "Location of acme server to use.%nLet's Encrypt Prod :%n@|bold " + LE_PROD_URL + "|@%nLet's Encrypt Staging :%n@|bold " + LE_STAGING_URL + "|@")
```

Comment:
Does it make sense to keep the prod/staging URLs here?

It might be kind of helpful if you are trying to figure out the correct URL for a different ACME implementation, but it's kind of redundant now.

Reply:
It doesn't....cleaned it up here 37aeed6